src/workerd/io/compatibility-date.capnp

```diff
@@ -1066,13 +1066,15 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
      $compatEnableFlag("enable_nodejs_perf_hooks_module")
      $compatDisableFlag("disable_nodejs_perf_hooks_module")
      $experimental;
+     # $impliedByAfterDate(name = "nodeJsCompat", date = "2025-10-01");
    # Enables the Node.js perf_hooks module. It is required to use this flag with
    # nodejs_compat (or nodejs_compat_v2).
 
   enableGlobalPerformanceClasses @123 :Bool
      $compatEnableFlag("enable_global_performance_classes")
      $compatDisableFlag("disable_global_performance_classes")
-     $experimental;
+     $compatEnableDate("2025-10-15")
+     $impliedByAfterDate(name = "enableNodeJsPerfHooksModule", date = "2025-10-15");
```

Comment:
As I mentioned in the comment, having both of these on the same date is redundant. The `impliedByAfterDate` is only necessary if that implied by date is strictly *before* the `compatEnableDate`. If the date is going to be the same, then drop the `impliedByAfterDate` annotation.